PancakeSwap-Bots/Prediction-Admin-Operatons/src/findings.ts

```diff
@@ -0,0 +1,91 @@
+import { Finding, FindingSeverity, FindingType, LogDescription } from "forta-agent";
+
+type FindingGenerator = (log: LogDescription) => Finding;
+
+const pause: FindingGenerator = (log: LogDescription): Finding =>
+  Finding.fromObject({
+    name: "PancakePredictionV2 Operations",
+    description: "Pause event emitted",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    alertId: "CAKE-9-1",
```

Comment:
```suggestion
    protocol: "PancakeSwap"
    alertId: "CAKE-9-1",
```

Protocol field missing (also needs to be added to the rest of the FindingGenerators)